src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  --global-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__items {
+  margin-left: 20px;
+}
+
+.nav__items:first-child {
+  margin-left: 0;
+}
+
+.nav__link:link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  height: 60px;
+  align-items: center;
+  cursor: pointer;
+  transition: color 0.3s;
+}
+
+.nav__link:visited {
+  color: #000;
+}
+
+a.nav__link.is-active,
+.nav__link:hover,
+.nav__link:active {
+  color: var(--global-color);
+}
```

Comment:
Mind @Moroz-Dmytro comment - you are too specific with this selector

Probably your problem is that you are setting the color with `.nav__link:link` - it is more specific than `.is-active`

You may need set the color on a higher level, or with less specific selector